oauth2client/client.py

```diff
@@ -134,6 +134,13 @@ class AccessTokenRefreshError(Error):
     """Error trying to refresh an expired access token."""
 
 
+class HttpAccessTokenRefreshError(AccessTokenRefreshError):
+    """Error (with HTTP status) trying to refresh an expired access token."""
+    def __init__(self, *args, **kwargs):
+        super(HttpAccessTokenRefreshError, self).__init__(*args)
```

Comment:
It [inherits][1] via `AccessTokenRefreshError -> Error -> Exception` and neither subclass actually modifies the parent (beyond the name). I can confirm that `Exception` rejects `kwargs`:

```python
>>> Exception(a=10)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: exceptions.Exception does not take keyword arguments
```

---

Unfortunately, if you want to pass `*args`, you use named `kwargs` so this is the only way to go.

[1]: https://github.com/google/oauth2client/blob/29347a91356a958f500b2c59ae3ed85387c282bb/oauth2client/client.py#L125-L134